### ISSUE TITLE
Compose Multiple Styles

### ISSUE THREAD

**Issue by superstructor** [dhleong/spade#15]
Is it possible to compose multiple styles ? E.g.

```clojure
(defclass example
  []
  {:composes [(shadow) (color-3) (big-text)]})
```

**Comment by dhleong** [comment]
Yes! You can compose multiple styles. I just [updated the readme](26d4a4f319b8d58af878c51854542ef447abba9d) to clarify that you can either provide a single name or a collection of names.

**Comment by ouvanous** [comment]
Hi, 
I have some trouble using composes with a collection. 
Using something like `:composes [(style1) (style2)]` give an "Invalid argument to :composes key:"
I did try to use `:composes (seq [(style1) (style2)])` but then the name of the style that use compose is split as a collection 

I think the problem come from here : 
```
(->> (if (seq? composed)
           (into composed style-name)
           [composed style-name])
```

Using :composes `(seq [(style1) (style2)])` if should be: 
          ` (into composed [style-name])`
 Or using` :composes [(style1) (style2)]`
it should be `(flatten [composed style-name])`

Or maybe I simply miss something here 

Thanks for spade! 
Samuel

**Comment by dhleong** [comment]
@ouvanous That does look like a bug! `(conj composed style-name)` should also work. I'll try to find time to fix this, but PR is welcome in the meantime ;)

**Comment by dhleong** [comment]
@ouvanous I just pushed a fix; should be available in the `1.2.0-SNAPSHOT`. Let me know if that resolves the issue for you!

**Comment by dhleong** [comment]
I went ahead and published a stable 1.1.1 with the fix
